tensorbay/client/version.py

```diff
@@ -5,31 +5,29 @@
 
 """TensorBay dataset version control related classes."""
 
-from typing import TYPE_CHECKING, Any, Dict, Generator, Optional, Union
+from typing import Any, Dict, Generator, Optional, Union
 
+from tensorbay.client.job import SquashAndMergeJob
 from tensorbay.client.lazy import PagingList
+from tensorbay.client.requests import Client
 from tensorbay.client.status import Status
 from tensorbay.client.struct import Branch, Commit, Draft, Tag
 from tensorbay.exception import ResourceNotExistError, StatusError
 
-if TYPE_CHECKING:
-    from tensorbay.client.gas import GAS
 
-
-class VersionControlClient:
-    """TensorBay dataset version control client.
+class VersionControlMixin:  # pylint: disable=too-many-public-methods
```

Comment:
I think it will be better to put `VersionControlMixin` releated code into another PR